tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3784,8 +3853,52 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        computeClosestCounterfactual: function(exInd, distances) {
```

Comment:
readability nit: prefix `compute` means something in Polymer and it often means a function that is side-effect-free. Can you please rename it?

Reply:
Didn't know, thanks. Renamed it.